api/proto/teleport/legacy/types/types.proto

```diff
@@ -6347,6 +6347,20 @@ message PluginOktaSettings {
 
   // Sync settings controls the user and access list sync settings for Okta.
   PluginOktaSyncSettings sync_settings = 4;
+
+  // CredentialsInfo contains information about the Okta credentials.
+  PluginOktaCredentialsInfo credentials_info = 5;
+}
+
+// Defines settings for the SCIM plugin.
+message PluginOktaCredentialsInfo {
+  option (gogoproto.equal) = true;
+  // HasConfiguredSSMSToken is true if the plugin has configured SSMSToken.
+  bool hasConfiguredSSMSToken = 1;
+  // HasConfiguredOauthCredentials is true if the plugin has configured OauthCredentials.
+  bool hasConfiguredOauthCredentials = 2;
+  // HasConfiguredSCIMToken is true if the plugin has configured SCIMToken.
+  bool hasConfiguredSCIMToken = 3;
```

Comment:
Shouldn't be snake_case?

Reply:
Yes, In this file we have mix of CamelCase and snake_case but snake_case where snake_case is suggested for google proto. 
Fixed.